.github/workflows/webui-periodic.yml

```diff
@@ -0,0 +1,107 @@
+# Run webui e2e tests periodically in Permian
+name: webui-periodic
+on:
+  schedule:
+    - cron: 0 23 * * *
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  scenario:
+    name: Webui periodic
+    runs-on: [self-hosted, kstest-permian]
+
+    timeout-minutes: 65
+    env:
+      # ocp-master-xxl: 32GB RAM / 4GB RAM per VM
+      TEST_JOBS: 8
+      # The timeout should be a few minutes less then the job's timeout-minutes
+      # so that we get partial results and logs in case of the timeout.
+      LAUNCHER_TIMEOUT_MINUTES: 60
+      # URL to the unpacked installation image
+      # TODO: use preview image
+      STATIC_REPO_URL: "https://fedorapeople.org/groups/anaconda/webui_permian_tests/sources/periodic/x86_64/"
+
+    steps:
+      # self-hosted runners don't do this automatically; also useful to keep stuff around for debugging
+      # need to run sudo as the launch script and the container create root/other user owned files
+      - name: Clean up previous run
+        run: |
+          sudo podman ps -q --all --filter='ancestor=kstest-runner' | xargs -tr sudo podman rm -f
+          sudo podman volume rm --all || true
```

Comment:
I think this would remove squid once deployed to regular runners?

Reply:
I was in some doubts about need for this, but we are using the same for kickstart tests workflows. So if, I'd fix it globally then.

Reply:
Oh, then it probably works well enough, or we will find out the hard way. I was not sure about what volumes we have.